### PR TITLE
Fix 404 errors in footer on sign-in page

### PR DESCRIPTION
**Pull request by michaelwmcnamara**
This is a fix to https://github.com/guardian/frontend/issues/8336
On sign-in page links in the footer were returning 404 errors.
Reason for this was that the links in question were relative links and the base url of the sign-in page is https://profile.theguardian.com/signin not http://www.theguardian.com .

Change is to hardcode http://www.theguardian.com as the base url in all relative links in the footer.

### CODE REVIEW

File: common/app/views/fragments/footer.scala.html [hunk 1]
```diff
@@ -75,50 +75,50 @@
             }
 
             @if(currentEdition == Au) {
-                <li class="colophon__item"><a data-link-name="au : footer : advertising" href="/advertising/guardian-australia-advertising">
+                <li class="colophon__item"><a data-link-name="au : footer : advertising" href="www.theguardian.com/advertising/guardian-australia-advertising">
```
Comment (johnduffell) [inline common/app/views/fragments/footer.scala.html]: these should probably have http:// or at least // on the start

**Comment by johnduffell** [comment]
apart from that :+1: and congrats on a great first PR!

**Comment by michaelwmcnamara** [comment]
added those now :-) cheers!

**Comment by gidsg** [comment]
Is there a variable for the guardian host rather than hard coding it?

**Comment by marchibbins** [comment]
@michaelwmcnamara @johnduffell coming across a similar issue elsewhere, is there a reason you didn't use `@LinkTo`?

**Comment by johnduffell** [comment]
@marchibbins because we didn't know about it!  what should we have done?

**Comment by marchibbins** [comment]
@johnduffell no worries! It builds absolute links from relative paths, using the `guardian.page.host` property (you may need to set this locally). I've opened #8716 with the update.

**Pull request merged**